baxter_collaboration_lib/test/test_robot_arm.cpp

```diff
@@ -0,0 +1,93 @@
+#include <gtest/gtest.h>
+#include "robot_interface/arm_ctrl.h"
+
+using namespace std;
+
+
+    //Potentially expand on the testcases; check to see that all edge values have been covered
+
+TEST(ArmControlTest, testConstructorDefaultValues)
+{
+    std::string _name = "robot"; 
+    std::string _limb = "left";
+    bool _use_forces = true; 
+    //Since we are actually not using the robot, _use_robot needs to be set to 'false' even though by default it should be true
```

Comment:
split this in two lines. In general, try to avoid having long lines. The guideline says to use a max of 80 columns, but it is not mandatory to be always below 80 columns.